object/store_provider.go

```diff
@@ -125,7 +142,17 @@ func (store *Store) Populate(origin string) error {
 			return err
 		}
 
-		tokens := strings.Split(strings.Trim(object.Key, "/"), "/")
+		// Remove sub path prefix to get relative path
+		relativePath := store.removeSubPath(object.Key)
```

Comment:
The object.Key is already subpath-removed in ListObjects(), why do it again here?